custom_components/ha-glocaltokens/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-  "domain": "ha-glocaltokens",
-  "name": "HA-Glocaltokens",
```

Comment:
I was thinking of having domain and name a bit different from `glocaltokens`, since `glocaltokens` is standalone python package, where this one is HomeAssistant specific integration

Reply:
Especially now that we are expanding the integration to do more functionality

Reply:
my suggestion would be glocalhome

Reply:
I like it! @DurgNomis-drol, drop it here: https://github.com/leikoilja/ha-glocaltokens/issues/19

Reply:
Let's not rename it in this PR, but have a whole new PR to rename everything at one once we settle on the name

Reply:
Good idea